Sources/Workspace/Workspace.swift

```diff
@@ -905,12 +905,20 @@ public class Workspace {
         return try resolver.resolve(constraints: constraints)
     }
 
+    // FIXME: Temporary shim while we transition to the new methods.
+    public func loadDependencyManifests(
+        rootManifests: [Manifest],
+        engine: DiagnosticsEngine
```

Comment:
Nitpick: it would be good to call this parameter something more specific than "engine", ideally something involving diagnostics.  I realize that that name isn't new to this diff, but I think the comment is still valid.